### DISCUSSION TITLE
How to completely override a theme?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [NvChad/NvChad#1508] [Q&A] [answered]
I recently found the [`kanagawa.nvim`](https://github.com/rebelot/kanagawa.nvim) theme and I absolutely love it, I mean the original version. The NvChad version has a lot of red that I don't like and some of the colors are different. This applies to other NvChad themes too, most of them for some reason have more "red" than the original theme. 

I understand that re-creating every theme with only 16 or 46 colors is not possible. I'm not criticizing you (siduck), you did a great job so far, I'm just looking for a way to override the NvChad theme. I mean I want to install other themes and load them after the NvChad theme so the highlights get overridden. Is that possible?

**Comment by siduck** [comment, marked as answer]
https://nvchad.github.io/config/theming#override-specific-colors-in-themes, so just remove the extra red parts or improve the original theme. I can test kanagahyamn on vscode only! and fix the existing kanagamn theme in nvchad while looking at the og theme side by side

**Reply by amirhhashemi** [reply]
Thanks for the response. Instead of changing each color manually, I just loaded the theme after `base46` so all `kanagawa.nvim` highlights override `base64` highlights.

```lua
  ["rebelot/kanagawa.nvim"] = {
    after = "base46",
    config = function()
      vim.cmd("colorscheme kanagawa")
    end,
  },
```

**Reply by siduck** [reply]
hmm thats going to slow lots of things & override highlights defined by our base46 plugin which isnt good :l

**Reply by amirhhashemi** [reply]
But that's the easiest way I found. Overriding each color takes a lot of time. The original kanagawa is very different than the NvChad's one. Not just the red colors but everything is different.

**Reply by siduck** [reply]
i'll test it some day and correct our kanagawa then

**Reply by amirhhashemi** [reply]
That's great. Thank you. 
I've actually tried to improve your kanagawa theme before, but I failed. In my opinion, it's very hard to copy the original theme with your current 16-base color system. The repo name is `base46` but you are still using only 16 colors.

**Reply by siduck** [reply]
@ahhshm you didnt check the polish_hl table :p 

so basically you can add any number of colors but you need to define highlight groups in the polish_hl table of the theme file, check catppuccin for example

**Reply by amirhhashemi** [reply]
Interstring. That makes things so much easier. I can correct some of the colors today.
